example_test.go

```diff
@@ -311,3 +343,34 @@ func Example_copyArtifactManifestRemoteToLocal() {
 	// Output:
 	// true
 }
+
+func Example_copyArtifactManifestAndReferrersRemoteToLocal() {
+	src, err := remote.NewRepository(fmt.Sprintf("%s/source", remoteHost))
+	if err != nil {
+		panic(err)
+	}
+	dst := memory.New()
+	ctx := context.Background()
+
+	// tag exampleManifest as latest in the remote repository
+	exampleTag := "latest"
+	err = src.Tag(ctx, exampleManifestDescriptor, exampleTag)
+	if err != nil {
+		panic(err)
+	}
+	// copy this manifest and its referrer(exampleSignatureManifest) using ExtendedCopy
+	copiedDescriptor, err := oras.ExtendedCopy(ctx, src, exampleTag, dst, exampleTag, oras.DefaultExtendedCopyOptions)
+	if err != nil {
+		panic(err)
+	}
+	// verify that exampleManifest and its referrer exist in dst
+	fmt.Println(reflect.DeepEqual(copiedDescriptor, exampleManifestDescriptor))
+	manifestExists, _ := dst.Exists(ctx, exampleManifestDescriptor)
```

Comment:
We should not ignore the returned error.

Reply:
Deleted this entire part of checking.